include/mls/credential.h

```diff
@@ -47,23 +47,27 @@ struct X509Credential
   SignatureScheme _signature_scheme;
 };
 
+tls::ostream&
+operator<<(tls::ostream& str, const X509Credential& obj);
+
+tls::istream&
+operator>>(tls::istream& str, X509Credential& obj);
+
 struct UserInfoVCCredential
 {
   UserInfoVCCredential() = default;
-  explicit UserInfoVCCredential(bytes userinfo_vc_jwt);
+  explicit UserInfoVCCredential(bytes userinfo_vc_jwt_in);
 
   bytes userinfo_vc_jwt;
 
   bool valid_for(const SignaturePublicKey& pub) const;
 
   TLS_SERIALIZABLE(userinfo_vc_jwt)
-};
 
-tls::ostream&
-operator<<(tls::ostream& str, const X509Credential& obj);
-
-tls::istream&
-operator>>(tls::istream& str, X509Credential& obj);
+  private:
+  SignaturePublicKey _public_key;
+  SignatureScheme _signature_scheme;
```

Comment:
nit, s/_public_key/public_key and same applies to similar class properties to keep it uniform in the code base ?

Reply:
This is consistent with `X509Credential`.  We're not super consistent throughout the library, but generally private fields get an underscore prefix.